apps/src/aiComponentLibrary/suggestedPrompt/SuggestedPrompts.tsx

```diff
@@ -20,28 +19,35 @@ interface SuggestedPromptsProps {
 const SuggestedPrompts: React.FC<SuggestedPromptsProps> = ({
   suggestedPrompts,
 }) => {
-  const setValues = (selected: string[]) => {
```

Comment:
This made it look like we supported multiple selections, but in reality we were prompting the AI the moment a user clicks the first one.